utils/swift_build_support/tests/products/test_swift.py

```diff
@@ -62,6 +62,8 @@ def setUp(self):
             enable_experimental_concurrency=False,
             enable_experimental_distributed=False,
             build_swift_stdlib_static_print=False,
+            stdlib_unicode_data=True,
+            enable_experimental_string_processing=False,
```

Comment:
This line isn't showing up in my local git diff against main and I'm so confused heh